lightning/src/util/persist.rs

```diff
@@ -491,14 +499,45 @@ where
 	/// less frequent "waves."
 	///   - [`MonitorUpdatingPersister`] will potentially have more listing to do if you need to run
 	/// [`MonitorUpdatingPersister::cleanup_stale_updates`].
+	///
+	/// This sets `min_monitor_size_for_updates_bytes` to 8192 bytes (8 KiB), which is a reasonable
+	/// default for most use cases. Monitors smaller than this will be persisted in full rather than
+	/// using update-based persistence. Use [`MonitorUpdatingPersister::new_with_monitor_size_threshold`]
+	/// if you need a custom threshold.
 	pub fn new(
 		kv_store: K, logger: L, maximum_pending_updates: u64, entropy_source: ES,
 		signer_provider: SP, broadcaster: BI, fee_estimator: FE,
+	) -> Self {
+		Self::new_with_monitor_size_threshold(
+			kv_store,
+			logger,
+			maximum_pending_updates,
+			8192,
```

Comment:
Echoing @tnull's request to make this an appropriately named constant from a previous diff. https://github.com/lightningdevkit/rust-lightning/pull/3834#discussion_r2171165554

Reply:
added the named const.